### PR TITLE
added UI defaults options to AdminSettings

### PR DESCRIPTION
**Pull request by merttumer**
Signed-off-by: Mert Tumer <mert.tumer@collabora.com>

* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [x] Code is properly formatted
- [ ] Sign-off message is added to all commits
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Comment by merttumer** [comment]
This patch depends on unreleased feature, let's wait until it is released, thanks

File: src/components/AdminSettings.vue [hunk 1]
```diff
@@ -193,6 +193,29 @@
 				</div>
 			</settings-checkbox>
 		</div>
+		<div v-if="isSetup" id="ui-defaults" class="section">
+			<h2>{{ t('richdocuments', 'UI Defaults') }}</h2>
+			<settings-checkbox v-model="settings.uidefaults.enabled" :label="t('richdocuments', 'Override UI Defaults')" hint=""
+				:disabled="updating" @input="updateUIDefaults" />
```
Comment (juliushaertl) [inline src/components/AdminSettings.vue:199]: I actually don't think we should have individual settings for this and instead pick sane defaults, what do you think @jancborchardt

Comment (merttumer) [reply src/components/AdminSettings.vue:199]: loolwsd.xml will also have these options and this will override them if checked.

Comment (jancborchardt) [reply src/components/AdminSettings.vue:199]: Agree with @juliushaertl here, we shouldn't expose these as settings and simply set our defaults.

**Pull request closed without merging**